style/_base.scss

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  text-align: center;
```

Comment:
Em ko nên dùng text-align: center; vào body nhé. Vì nó sẽ ảnh hưởng đến toàn bộ site.
Em tên tạo 1 class chung có thuộc tính này và khi nào cần sẽ gọi class đó vào